scripts/tests/twister_blackbox/test_disable.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# Copyright (c) 2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+"""
+Blackbox tests for twister's command line functions
+"""
+
+import importlib
+import mock
+import os
+import pytest
+import sys
+
+from conftest import ZEPHYR_BASE, TEST_DATA, testsuite_filename_mock
+from twisterlib.testplan import TestPlan
+from twisterlib.error import TwisterRuntimeError
+
+
+class TestError:
+    TESTDATA_1 = [
+        (
+            os.path.join(TEST_DATA, 'tests', 'dummy', 'agnostic'),
+            ['qemu_x86']
+            # {
+            #     'selected_test_scenarios': 1,
+            #     'selected_test_instances': 3,
+            #     'skipped_configurations': 3,
+            #     'skipped_by_static_filter': 3,
+            #     'skipped_at_runtime': 0,
+            #     'passed_configurations': 0,
+            #     'failed_configurations': 0,
+            #     'errored_configurations': 0,
+            #     'executed_test_cases': 0,
+            #     'skipped_test_cases': 3,
+            #     'platform_count': 3,
+            #     'executed_on_platform': 1,
+            #     'only_built': 0
+            # }
+        ),
+    ]
+
+    @classmethod
+    def setup_class(cls):
+        apath = os.path.join(ZEPHYR_BASE, 'scripts', 'twister')
+        cls.loader = importlib.machinery.SourceFileLoader('__main__', apath)
+        cls.spec = importlib.util.spec_from_loader(cls.loader.name, cls.loader)
+        cls.twister_module = importlib.util.module_from_spec(cls.spec)
+
+    @classmethod
+    def teardown_class(cls):
+        pass
+
+    @pytest.mark.parametrize(
+        'test_path, test_platforms, expected',
```

Comment:
`TESTDATA_1` after the comment-out only has two parameters, while you declare three here. It will error out.